:sparkles: add plotly-resampler (#40)
ucg8j/awesome-dash: README.md

- [dash-bootstrap-components](https://dash-bootstrap-components.opensource.asidatascience.com/) - Layout engine, default styles and high-level components based on Bootstrap.
- [dash-uploader](https://github.com/np-8/dash-uploader) - Upload component for Dash. Supports large data files.
- [Dash Mantine Components](https://github.com/snehilvj/dash-mantine-components) - Collection of 40+ Dash components based on Mantine React Components library.

> - [plotly-resampler](https://github.com/predict-idlab/plotly-resampler) - Wrapper for plotly figures that adds data aggregation functionality (via a dash app), enabling to visualize large sequences.

Reviewer: ```suggestion
- [plotly-resampler](https://github.com/predict-idlab/plotly-resampler) - Wrapper for plotly figures that adds data downsampling (aggregating) functionality, enabling the visualization of large datasets.
```

Author: I agree with your suggestion!
